src/components/Profile/Motto/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { Check, PencilAlt } from '@styled-icons/fa-solid'
+import React, { useEffect, useState } from 'react'
+import { useForm } from 'react-hook-form'
+import FormTextArea from '../../Form/components/FormTextArea'
+import { StyledParagraph } from '../../StyledText/StyledParagraph'
+import { Edit } from '../styled'
+
+type Props = {
+	motto: string | undefined
+}
+
+export const Motto: React.FC<Props> = ({ motto }) => {
+	const [element, setElement] = useState<boolean>(true)
+
+	const [text, setText] = useState<string | undefined>('')
+
+	useEffect(() => {
+		setText(motto)
+	}, [motto])
+
+	const { register, handleSubmit } = useForm()
+
+	const changeText = (text: string | undefined) => setText(text)
+
+	return (
+		<>
+			{element ? (
+				<form onSubmit={handleSubmit(() => setElement(false))}>
+					<FormTextArea
+						register={register}
+						value={text}
+						placeholder="Napiš své motto"
```

Comment:
I can't see this placeholder unless I delete "suggested" motto "Člověk nežije proto, aby pracoval, ale pracuje, aby žil."

Reply:
Hm, but at first the user sees only the placeholder and then sees only his motto. we no longer need to see the placeholder. The proposed text is just an example of a motto.

But thanks to your comment, I realized that if the user entered some text (heslo, něco o mně atd.), it would be better to show the text than the textarea! The text area should only appear for the first time if nothing is filled. 🤔

I'll definitely take a look at that, good point 👍